Calculator.Tests/Calculator.Tests/LoginPageTests.cs

```diff
@@ -40,98 +40,100 @@ public void PositiveTest()
         [Test]
         public void IncorrectLoginTest()
         {
-            browser.FindElement(By.Id("login")).SendKeys("test1");
-            browser.FindElement(By.Id("password")).SendKeys("newyork1");
-            browser.FindElement(By.Id("loginBtn")).Click();
-            string actual = browser.FindElement(By.Id("errorMessage")).Text;
+            browser.FindElement(By.XPath("//input[@id = 'login']")).SendKeys("test1");
+            browser.FindElement(By.XPath("//input[@id = 'password']")).SendKeys("newyork1");
+            browser.FindElement(By.XPath("//button [@id = 'loginBtn']")).Click();
+            string actual = browser.FindElement(By.XPath("//tr[4]//th[contains(text(), 'Incorrect user name!')]")).Text;
 
             Assert.AreEqual("Incorrect user name!", actual);
         }
 
         [Test]
         public void IncorrectPasswordTest()
         {
-            browser.FindElement(By.Id("login")).SendKeys("test");
-            browser.FindElement(By.Id("password")).SendKeys("newyork11");
-            browser.FindElement(By.Id("loginBtn")).Click();
-            string actual = browser.FindElement(By.Id("errorMessage")).Text;
+            browser.FindElement(By.XPath("//input[@id = 'login']")).SendKeys("test");
+            browser.FindElement(By.XPath("//input[@id = 'password']")).SendKeys("newyork11");
+            browser.FindElement(By.XPath("//button [@id = 'loginBtn']")).Click();
+            string actual = browser.FindElement(By.XPath("//tr[4]//th[contains (text(), 'Incorrect password!')]")).Text;
 
             Assert.AreEqual("Incorrect password!", actual);
         }
 
         [Test]
         public void IncorrectLoginPasswordTest()
         {
-            browser.FindElement(By.Id("login")).SendKeys("test1");
-            browser.FindElement(By.Id("password")).SendKeys("newyork11");
-            browser.FindElement(By.Id("loginBtn")).Click();
-            string actual = browser.FindElement(By.Id("errorMessage")).Text;
+            browser.FindElement(By.XPath("//input[@id = 'login']")).SendKeys("test1");
+            browser.FindElement(By.XPath("//input[@id = 'password']")).SendKeys("newyork11");
+            browser.FindElement(By.XPath("//button [@id = 'loginBtn']")).Click();
+            string actual = browser.FindElement(By.XPath("//tr[4]//th[@id = 'errorMessage']")).Text;
 
             Assert.AreEqual("'test1' user doesn't exist!", actual);
+            //NEED HELP in line 68: I could not find it by //tr[4]//th[contains (text(), ''test1' user doesn't exist!')]
+            //is it because of 'test1' signs or what do you think?
```

Comment:
Because there is ['] symbol in text. As a solution you may try:
```suggestion
            string actual = browser.FindElement(By.XPath("//th[contains (text(), \"'test1' user doesn't exist!\"")).Text;
```

Reply:
Actually, I have read about '\' sign but could not use it properly. thanks!

string actual = browser.FindElement(By.XPath("//th[contains (text(), \"'test1' user doesn't exist!\"")).Text; 
this code does not find the element, and also I see such mistake while debug run
<img width="591" alt="Screenshot 2021-04-06 at 09 44 09" src="https://user-images.githubusercontent.com/57406734/113669910-17f74b00-96bd-11eb-9d26-60c8f4a004f8.png">

Reply:
string actual = browser.FindElement(By.XPath("//th[contains (text(), \"'test1' user doesn't exist!\"")).Text;

Reply:
inner quotes should be 'escaped':
```
\"
```

Reply:
I see, thank you. I rewrote this particular test to use By.Id 

That code which you gave `string actual = browser.FindElement(By.XPath("//th[contains (text(), \"'test1' user doesn't exist!\"")).Text;` also does not want to work ((